addons/website/views/snippets/s_media_list.xml

```diff
@@ -2,57 +2,75 @@
 <odoo>
 
 <template name="Media List" id="s_media_list">
-    <section class="s_media_list bg-200 pt24 pb24">
+    <section class="s_media_list bg-200 pt32 pb32" data-vcss="001">
         <div class="container">
-            <div class="row" data-name="Media item">
-                <div class="col-lg-4">
-                    <img class="img img-fluid" src="/web/image/website.s_media_list_default_image_1"  alt=""/>
+            <div class="row s_nb_column_fixed s_col_no_bgcolor">
+                <div class="col-lg-12 s_media_list_item pt16 pb16" data-name="Media item">
```

Comment:
Is the first level of row -> col useful in fact?

Reply:
Yes. All the options related to this snippet appears at the first level. You can also resize/move/delete the entire block from here.

Reply:
I understand that but the what do you get from it being a row > col? The resizing of the whole media item seems to be the only use case and I am not sure it is a good one? But if you find it useful, I guess this does not hurt ;)